### ISSUE TITLE
softmax

### ISSUE THREAD

**Issue by ardeal** [Xilinx/brevitas#404]
Hi,

Is softmax supported?
how to use it in my python code?
Is there any document about sofmax in Brevitas?

**Comment by MohamedA95** [comment]
AKAIK there is no quantized softmax layer but you can use it from torch it's just a mathematical layer.

**Comment by ardeal** [comment]
@MohamedA95 ,
Thank you!
To use `torch.nn.Softmax()` in code is not difficult, my question is:
the output of the upper layer(`qnn.QuantLinear`) is quantized data, could it be passed to `torch.nn.Softmax()` directly?

**Comment by MohamedA95** [comment]
Hi @ardeal  yes it could be passed directly.
